modules/service/src/main/scala/ch/epfl/bluebrain/nexus/admin/service/routes/ProjectAclRoutes.scala

```diff
@@ -34,14 +35,14 @@ final class ProjectAclRoutes(projects: Projects[Future], proxy: Proxy)(implicit
   private val iamUri = config.iam.baseUri
 
   protected def combined(implicit credentials: Option[OAuth2BearerToken]): Route =
-    (segment2(of[ProjectReference]) & pathPrefix("acls") & pathEndOrSingleSlash) { name =>
+    (projectReference & pathPrefix("acls") & pathEndOrSingleSlash) { name =>
       (exists(name) & parameterMap & extractRequest) { (params, req) =>
         trace(s"${req.method.toString().toLowerCase()}ProjectACL") {
           complete(
             proxy(
               req
                 .withHeaders()
-                .withUri(iamUri.append("acls" / name.value).withQuery(Query(params)))
+                .withUri(iamUri.append("acls" / name.show).withQuery(Query(params)))
```

Comment:
Is the `append` method, when passing a string with a slash, treating as a Path.Slash or it is URL encoding the slash because it thinks it belongs to the segment? I'm talking about the part on `name.show`